test/es6/module-syntax.js

```diff
@@ -8,7 +8,7 @@
 WScript.LoadScriptFile("..\\UnitTestFramework\\UnitTestFramework.js");
 
 function testModuleScript(source, message, shouldFail) {

```

Comment:
>shouldFail [](start = 43, length = 10)

relying on default value = undefined? could explicitly have `shouldFail = false` instead to be more explicit about this.